app/src/main/java/at/aau/serg/activities/CreateLobbyActivity.kt

```diff
@@ -0,0 +1,81 @@
+package at.aau.serg.activities
+
+import android.content.Intent
+import android.os.Bundle
+import android.util.Log
+import android.view.View
+import android.widget.CheckBox
+import android.widget.EditText
+import android.widget.Toast
+import androidx.activity.enableEdgeToEdge
+import androidx.appcompat.app.AppCompatActivity
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import at.aau.serg.R
+import at.aau.serg.logic.StoreToken
+import at.aau.serg.models.LobbyCreate
+import at.aau.serg.network.CallbackCreator
+import at.aau.serg.network.HttpClient
+import com.google.android.material.slider.Slider
+import com.google.gson.Gson
+import okhttp3.Response
+
+class CreateLobbyActivity : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        enableEdgeToEdge()
+        setContentView(R.layout.activity_create_lobby)
+        ViewCompat.setOnApplyWindowInsetsListener(findViewById(R.id.main)) { v, insets ->
+            val systemBars = insets.getInsets(WindowInsetsCompat.Type.systemBars())
+            v.setPadding(systemBars.left, systemBars.top, systemBars.right, systemBars.bottom)
+            insets
+        }
+    }
+
+    fun btnMainActivity(view: View) {
+        startActivity(Intent(this, MainActivity::class.java))
+    }
+
+    fun btnCreateLobby(view: View) {
+        cLobby(null)
+    }
+
+    private fun cLobby(response: Response?) {
```

Comment:
We could set a default value here so that we don't have to pass null in `btnCreateLobby`

Reply:
I would do that in the pr related to the callbackcreater and the error handling of it